### ISSUE TITLE
Easier way to change theme

### ISSUE THREAD

**Issue by greenbech** [zeit/hyper#2942]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS 10.13.4
- **Hyper.app version**: Hyper 2.0.0 (stable)
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: Default
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla
## Issue

I find it a bit cumbersome to change themes in Hyper. Is there a better way than 
- first open `.hyper.js`,
- find `plugins: ` and then possibly comment out your old theme and then 
- write the name of the new theme you want that you found somewhere on the web? 

It would be nice to have some sort of shortcut and possibly have a option under `view` that open a _grafical theme picker_ that resembled https://hyper.is/themes and when you clicked a theme it automatically changed in `.hyper.js` and the theme was updated. I believe this would made the transition to Hyper easier for most people!

**Comment by albinekb** [comment]
You can type `hyper i <theme name>` to install/preview a theme 👍

**Comment by greenbech** [comment]
But that still dosen't solve the problem that you have to comment/remove/uninstall the last theme you had. See picture for what happened when I first installed `"hyper-solarized-light"` then `"hyper-one-light"`. 

`hyper i <theme name>` should probably comment out or remove all the other themes in `plugins`.

`plugins: ["hyper-solarized-light", "hyper-one-light"]`:
![plugins: ["hyper-one-light", "hyper-one-light"]](https://user-images.githubusercontent.com/7979907/39376165-fa0fcd6c-4a50-11e8-9c4a-3f107149d346.png)

For reference:
"hyper-one-light"
![skjermbilde 2018-04-27 19 28 42](https://user-images.githubusercontent.com/7979907/39376406-e3ee0bf6-4a51-11e8-93ea-57565c61d198.png)

"hyper-solarized-light"
![skjermbilde 2018-04-27 19 28 27](https://user-images.githubusercontent.com/7979907/39376439-00968922-4a52-11e8-9e18-0f2a43ea12d6.png)

**Comment by rosejn** [comment]
Yeah, this doesn't make sense to me either.  There is a difference between installing a theme or plugin and choosing which one to use.  If you install a few themes, which you will naturally do when trying some out, then there should be an equivalently straight forward way to select one.  Maybe something like:

hyper theme hyper-solarized-light

**Comment by devniel** [comment]
Had this same problem, I was checking themes, now I want to use specifically one but there is no direct way to do it. 

I tried `hyper i verminal` again but I got the `verminal is already installed` error.

Thanks.

**Comment by apjanco** [comment]
I agree that a command to change the theme is needed.  For now, you can uninstall `hyper u <theme-name>` and then re-install `hyper i <theme-name>`.

**Comment by fooski** [comment]
I second this, need a much easier way to change theme

**Comment by mihirkumar** [comment]
This was opened over a year ago. Has there been any development on this issue? It's central to Hyper's usage 😞 I like the project but changing themes is so cumbersome.

**Comment by Stanzilla** [comment]
@mihirkumar nope, sorry. the original authors and maintainers of the project all left, PRs welcome though.

**Comment by iamjaredwalters** [comment]
> the original authors and maintainers of the project all left
@Stanzilla is this in reference to the theme switching functionality described in this issue or the Hyper project as a whole?

**Comment by phrazzld** [comment]
Keeping a commented list in my `.hyper.js` file isn't terrible, but it isn't great. 

`hyper theme <theme-name>` would be ideal, using `<theme-name>` if it's installed, or installing and using it if it is not.

**Comment by rdewolff** [comment]
Maybe that means refactoring the config file. We could have a dedicated propery `themes: []` so we clearly separate them from `plugins: []`. Then either the first theme of the array would be the active one, or using an additional property `activeTheme`. 

What do you think?

**Comment by ppot** [comment]
This is something I vouched for some times. Will require a bigger refactor in order to do so. Feel free to propose something.

**Comment by GeoBrodas** [comment]
OK guys I finally found out how to change the theme on hyper I don't know whether this fix was declared but here it is I figured it out myself
So what hyper does is if you have installed a particular theme already its shows `theme already installed` and there is no provision as such to select between themes...so here's what you can do

> Go to preferences in edit section
> Click preferences
> It'll open up a json file I suppose that's because hyper is built using Electron
> Scroll down to plugins
> Clear the theme you want to set if its shown in the list and save the file
> Go back tot erminal and install it normally!
It'll work!

Alhough this is a manual proccedure it would be better if Hyper provides us to select between themes.
Thankyou!

**Comment by edisim** [comment]
@GeoBrodas as @apjanco mentioned before, you could just type `hyper u <theme-name> && hyper i <theme-name>` to reinstall it, which seems to be a faster approach

**Comment by GeoBrodas** [comment]
@edisim `hyper i ` does'nt work I have tried it 
will try `hyper u` as you suggested

**Comment by SynergyBest** [comment]
> @edisim `hyper i ` does'nt work I have tried it
> will try `hyper u` as you suggested

you have to install the cli tool
<img width="412" alt="Screen Shot 2021-03-01 at 1 18 47 PM" src="https://user-images.githubusercontent.com/72149158/109547218-bb3abc00-7a90-11eb-8893-f35f3c47afa4.png">
`plugins >> install Hyper CLI in PATH`

**Comment by SynergyBest** [comment]
didn't mean to mention you edism, my bad 😬

**Comment by ng-hai** [comment]
Hi, I have just released a theme that supports Dark Mode. It worked on Hyper Canary.
https://github.com/ng-hai/hyper-rose-pine-next


https://user-images.githubusercontent.com/19520278/119014347-15795800-b9c2-11eb-9557-8a33086c5413.mov

**Comment by tbremer** [comment]
I was wanting a simple light/dark mode theme switcher and was inspired by @ng-hai and released a package that allows for just that (themes used here are [hyper-papercolor](https://github.com/rafaelrinaldi/hyper-papercolor/) & [hyper-oceans16](https://github.com/henrikdahl/hyper-oceans16)).

https://github.com/tbremer/hyper-prefers-color-scheme


https://user-images.githubusercontent.com/2998575/130810231-0292bc59-ae7c-4b8d-a5fc-fb47f8faae65.mp4

**Comment by omysandbag** [comment]
Hello,

Is it possible to change the theme per tab or window?

This proposes when I work with multiple cluster servers via ssh and the theme would help separate them by color.

Thank you.

**Comment by vPackets** [comment]
Yep I agree, would be awesome to just type "hyper theme <name>" and voila !

**Comment by Yb2S3Man** [comment]
I highly suggest to have a command to switch the theme. Use u/i is not good idea which seems too naive.

**Comment by chillyspoon** [comment]
A UI feature for this is really lacking - given that the array is already there in the preferences json indicates that the intention was to support multiple themes with an ability to flip between them but it straight up doesn't work properly so I ended up here looking at a 6 year old feature request when I couldn't figure out from the website themes page how to quickly test drive and compare themes apart from re-installation, and ended up searching for answers on the interweb - please add a menu item for this bit of QoL. It would make an already great terminal even greater!
